doc/classes/GPUParticles2D.xml

```diff
@@ -50,6 +50,10 @@
 		<member name="amount" type="int" setter="set_amount" getter="get_amount" default="8">
 			Number of particles emitted in one emission cycle.
 		</member>
+		<member name="amount_ratio" type="float" setter="set_amount_ratio" getter="get_amount_ratio" default="1.0">
+			The ratio of particles that should actually be emitted. If set to a value lower than [code]1.0[/code], this will set the amount of emitted particles throughout the lifetime to [code]amount * amount_ratio[/code]. Unlike changing [member GPUParticles2D.amount], changing [member amount_ratio] while emitting does not affect already-emitted particles and doesn't cause the particle system to restart. [member amount_ratio] can be used to create effects that make the number of emitted particles vary over time.
+			[b]Note:[/b] If you don't need to change the number of particles emitted while particles are emitting, keep [member amount_ratio] at [code]1.0[/code] and change [member GPUParticles2D.amount] instead. This will result in better performance, as inactive particles still need to be partially evaluated in the shader otherwise.
```

Comment:
This seems overexplained, imo make it more user-facing like

```suggestion
			[b]Note:[/b] Consider changing the [member amount] instead if the particle system restarting is not a problem, as changing the [member amount_ratio] is more performance-intensive.
```

Also change [member GPUParticles2D.amount] to just [member amount] in the description right above, and also for 3D.

Reply:
I am not sure for either formulation. I am considering actually removing the note since I am hoping to move the code that calculates that out of the shader, so there shouldn't be any performance impact.

Maybe could change the note to:

Note: reducing the amount ratio doesn't cause any increase in performance, since the number of allocated rendering resoures depends on amount.

What do you think?

Reply:
If there's no performance concern people should be aware of, I wouldn't leave a note.

Reply:
Oh, makes sense. I think your description is fine then, just add bold and member references.

Reply:
Bold? Where?

Reply:
Haha you know, [b]Note[/b]. Sorry for confusing ya.

Reply:
But it's already there? Or not? Sorry I'm very very confused